README.md

```diff
@@ -30,165 +32,557 @@ The two guiding principles are:
+
+## System requirements
+
+Hops is built on modern technologies and therefore needs at least Node.js _v8.10_ or higher.
+
+You can use either [`npx`](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b) (which is included in npm v5.2+) or [`npm init`](https://docs.npmjs.com/cli/init) (with npm v6+) or [`yarn create`](https://yarnpkg.com/lang/en/docs/cli/create/) to create a Hops application and run Hops CLI commands.
+
 ## Quick start
 
-Install the `hops` CLI globally:
+To create a new Hops application run the following in a terminal:
+
+```shell
+npx create-hops-app my-hops-app
+```
+
+_Note: If you prefer to use `yarn`, you can substitute the above command with `yarn create hops-app my-hops-app`._
+
+Then move into the newly created directory:
 
 ```shell
-$ yarn global add hops
-# OR npm install --global hops
+cd my-hops-app
 ```
 
-Or use our [`create-hops-app`](https://github.com/xing/hops/packages/create-hops-app) package with [`yarn create`](https://yarnpkg.com/lang/en/docs/cli/create/) or [`npx`](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b):
+And start the development server:
 
 ```shell
-$ yarn create hops-app@next --hops-version next --template hops-template-react@next my-awesome-project
-# OR npx create-hops-app@next --hops-version next --template hops-template-react@next my-awesome-project
-$ cd my-awesome-project
-$ yarn start
+npm start
 ```
 
-This will start hops in development mode. Visit [http://localhost:8080](http://localhost:8080) to see your app in the browser and make some changes to the code in your editor to see it live-reloading.
+This will start Hops in development mode. Visit [http://localhost:8080](http://localhost:8080) to see your app in the browser and make some changes to the code in your editor to see it live-reloading.
 
-### Templates
+## Walk-through
 
-Hops provides a few templates as starting points, which can be installed using the `--template` option of the [Hops CLI `init` command](https://github.com/xing/hops/tree/master/packages/cli).
+### Entry files and `hops`'s `render()` function
 
-#### [`hops-template-react`](https://github.com/xing/hops/tree/master/packages/template-react) (default)
+Hops assumes your entry file is either named `index.js` or is specified via the [`"main"` field in your `package.json`](https://docs.npmjs.com/files/package.json#main). This is similar to how Node.js would resolve a package and allows you to place you entry file wherever you like.
 
-This is the default template, it consists of a simple React starting point and it is being used when you execute:
+**`package.json`**
 
-```bash
-$ hops init --hops-version next --template hops-template-react@next my-awesome-project
+```json
+{
+  "main": "./src/app.js"
+}
 ```
 
-#### [`hops-template-redux`](https://github.com/xing/hops/tree/master/packages/template-redux)
+The main export of the entry file must be the result of `hops`'s [`render()`](#render) function:
```

Comment:
This is supposed to link to `#renderelement-options`, right? `#render` does not exist. There are several occurrences throughout this file.